src/WattleScript.Interpreter/Tree/Statements/ClassDefinitionStatement.cs

```diff
@@ -77,23 +80,48 @@ public ClassDefinitionStatement(ScriptLoadingContext lcontext) : base(lcontext)
             initClosure = new GeneratedClosure($"{className}.__init", defSource, FunctionFlags.None, false);
             //__tostring()
             tostringClosure = new GeneratedClosure($"{className}.__tostring", defSource, FunctionFlags.TakesSelf, false);
+   
+            MemberModifierFlags modifierFlags = MemberModifierFlags.None;
+            
+            void AddMemberFlag(MemberModifierFlags flag)
+            {
+                if ((modifierFlags & flag) != 0)
```

Comment:
General note: It might be cleaner to use `modifierFlags.HasFlag(flag)` throughout this file

Reply:
resolved in bb8e857